test/template_t.cpp

```diff
@@ -83,6 +84,17 @@ QPointF center(const Template* temp)
 	return temp->templateToMap(temp->getTemplateExtent().center());
 }
 
+void center(const Template* temp, QPointF &center_point, QDateTime &center_time)
```

Comment:
Do you really want to deal with `center_time`? Rather than adding another overload of `center`, changing `center(const TemplateTrack* temp)`, but leaving `templateTrackTest`/`ogrTemplateTest` mostly unchanged, would look much clearer.

Reply:
I thought `center_time` would be needed, but the test doesn't use it, not even for #1757. I will simplify.

Reply:
Simplified as suggested. The change even removed `center(TemplateTrack *)` from `template_t.cpp`. The change has been pushed.